qttypes/src/lib.rs

```diff
@@ -640,23 +641,105 @@ impl std::fmt::Debug for QString {
     }
 }
 
+// XXX: The Default, Clone and Copy traits are provided by cpp_class! but they won't have these
+// nice "Wrapper around ..." docs with links on them. How can we fix this?
 cpp_class!(
     /// Wrapper around [`QVariant`][class] class.
     ///
     /// [class]: https://doc.qt.io/qt-5/qvariant.html
     #[derive(PartialEq)]
     pub unsafe struct QVariant as "QVariant"
 );
+// methods are ordered as in the official documentation
 impl QVariant {
-    /// Wrapper around [`toByteArray()`][method] method.
+    /// Wrapper around [`canConvert(int targetTypeId)`][method] method.
     ///
-    /// [method]: https://doc.qt.io/qt-5/qvariant.html#toByteArray
-    pub fn to_qbytearray(&self) -> QByteArray {
-        cpp!(unsafe [self as "const QVariant*"] -> QByteArray as "QByteArray" {
-            return self->toByteArray();
+    /// [method]: https://doc.qt.io/qt-5/qvariant.html#canConvert
+    pub fn can_convert(&self, target_type_id: i32) -> bool {
+        cpp!(unsafe [self as "const QVariant*", target_type_id as "int"] -> bool as "bool" {
+            return self->canConvert(target_type_id);
+        })
+    }
+
+    // TODO: for this to work, we need QMetaType be available in qttypes package.
+    // TODO: It would create a dependency cycle at this point.
```

Comment:
I think we don't need to expose this function in this crate.

The function in the QMetaType trait should be enough.

Reply:
OK